src/app/session/[type]/Steps/helper.ts

```diff
@@ -135,11 +137,15 @@ export const setParentBatchOptions = (
   fieldsSchema: FieldSchema<basicFields>
 ) => {
   if (!value) return;
-
   const authGroupId = apiOptions.group?.find((item) => item.value === value)?.id;
-  const filteredQuizBatchOptions = apiOptions?.batch?.filter(
-    (item) => item.groupId === authGroupId && !item.parentId
-  );
+  let filteredQuizBatchOptions: ExtendedOptions[] = [];
+  if (authGroupId === 14) {
```

Comment:
can u comment this is for tnschools

Reply:
infact we dont know if authgroupid will always be 14.. maybe a better way is to GET authgroupid given auth group name

Reply:
Should I Handle this check using Name?

Reply:
Done